examples/job/adsorption-study.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[]:
+
+
+import os, sys
+import string
+
+import ase.io
+import ase.neighborlist
+import ase.constraints
+
+import numpy as np
+
+# Import settings file and utils
+module_path = os.path.abspath(os.path.join('..'))
+if module_path not in sys.path:
+    sys.path.append(module_path)
+from utils import ensure_packages_are_installed
+#ensure_packages_are_installed()
+from material_utils import download_contcar
+from settings import ENDPOINT_ARGS, ORGANIZATION_ID
+
+# Import relevant portions of the API client
+from exabyte_api_client.endpoints.jobs import JobEndpoints
+from exabyte_api_client.endpoints.projects import ProjectEndpoints
+from exabyte_api_client.endpoints.materials import MaterialEndpoints
+from exabyte_api_client.endpoints.workflows import WorkflowEndpoints
+from exabyte_api_client.endpoints.bank_workflows import BankWorkflowEndpoints
+
+
+# # Query the Platform for the Slab
+# 
+# Get slab, acquire science
+
+# In[]:
+
+
+job_endpoint = JobEndpoints(*ENDPOINT_ARGS)
+material_endpoint = MaterialEndpoints(*ENDPOINT_ARGS)
+
+# ToDo: Refactor class and add documentation
```

Comment:
Let's actually do what this ToDo note is calling for.

Reply:
Added documentation for the class, removed the material attribute since it was un-used elsewhere